internal/component/discovery/process/discover.go

```diff
@@ -167,3 +180,17 @@ func getLinuxProcessContainerID(pid string) (string, error) {
 	}
 	return "", nil
 }
+
+func getLinuxProcessCgroupID(pid string, regexp *regexp.Regexp) (string, error) {
+	if runtime.GOOS == "linux" {
```

Comment:
Is this needed?  With the go build linux flag at the top of the file.

Reply:
Makes sense. Will remove it.